osu.Game.Tests/ImportTest.cs

```diff
@@ -17,7 +17,8 @@ public abstract class ImportTest
         protected virtual TestOsuGameBase LoadOsuIntoHost(GameHost host, bool withBeatmap = false)
         {
             var osu = new TestOsuGameBase(withBeatmap);
-            Task.Run(() => host.Run(osu));
+            Task.Run(() => host.Run(osu))
+                .ContinueWith(t => Assert.Fail($"Host threw exception {t.Exception}"), TaskContinuationOptions.OnlyOnFaulted);
```

Comment:
Am I correct in thinking that this doesn't resolve the CI deadlocks?

Reply:
Yep, just something I found in passing (potentially will give us hints in such scenarios)